join-orgs-beta.js

```diff
@@ -0,0 +1,25 @@
+var tap = require('tap');
+var urlOf = require('./lib/url');
+var P = require('bluebird');
+
+require('./lib/sharedNemo').then(function(nemo) {
+  nemo.state.desiredUsername = 'test-org-' + Date.now();
+
+  require('./signup');
```

Comment:
Because these bail out if they fail, we can use them as building blocks and we'll fail if a subtest fails.